app/index.ts

```diff
@@ -17,33 +17,31 @@ const SETTINGS_FILE = "settings.json";
 let settings = loadSettings();
 applyTheme(settings.foreground);
 
-function updateClock(evt) {
+clock.ontick = evt => {
   let today = evt.date;
-  let hours = today.getHours();
-  if (preferences.clockDisplay === "12h") {
-    // 12h format
-    hours = (hours + 24) % 12 || 12;
-  }
-  else {
-    // 24h format
-    hours = util.zeroPad(hours);
-  }
+  const hours = today.getHours();
+  const strHours =
+    preferences.clockDisplay === "12h"
+      ? // 12h format
+        (hours + 24) % 12 || 12
+      : // 24h format
+        util.zeroPad(hours);
   let mins = util.zeroPad(today.getMinutes());
-  clockLabel.text = `${hours}:${mins}`;
-  clockShadow.text = `${hours}:${mins}`;
-}
-
-clock.ontick = (evt) => updateClock(evt);
+  clockLabel.text = `${strHours}:${mins}`;
+  clockShadow.text = `${strHours}:${mins}`;
+};
 
 messaging.peerSocket.onmessage = function(evt) {
   if (evt.data && evt.data.foreground) {
     applyTheme(evt.data.foreground);
   }
-}
+};
 
-function applyTheme(foreground) {
+function applyTheme(foreground: string) {
   display.on = true;
-  let items = document.getElementsByClassName("foreground");
+  let items = document.getElementsByClassName("foreground") as Array<
+    GraphicsElement
```

Comment:
Without this, the following `.style` sentences don't work because they're not present in `Element` but in at least `GraphicElement`.